test/sci/core_test.cljc

```diff
@@ -1581,6 +1581,11 @@
 (deftest api-resolve-test
   (is (= 2 ((sci/resolve (sci/init {}) 'clojure.core/inc) 1))))
 
+(deftest print-namespace-maps-var-test
```

Comment:
I think this test is a bit obsolete since we already have the other one

Reply:
OK. I shall remove it.